shell/platform/embedder/embedder.cc

```diff
@@ -2050,3 +2050,54 @@ FlutterEngineResult FlutterEngineNotifyDisplayUpdate(
           "Invalid FlutterEngineDisplaysUpdateType type specified.");
   }
 }
+
+FlutterEngineResult FlutterEngineGetProcAddresses(
```

Comment:
If the `table` argument is nullptr, return `kInvalidArgument`.

Reply:
Done.